packages/bpk-component-calendar/src/Week.js

```diff
@@ -205,39 +205,41 @@ class Week extends Component {
 
     return (
       <tr className={getClassName('bpk-calendar-grid__week')}>
-        {this.props.dates.map(date => (
-          <DateContainer
-            className={this.props.cellClassName}
-            isEmptyCell={!isSameMonth(date, month) && ignoreOutsideDate}
-            key={date.getDate()}
-            weekendStart={
-              showWeekendSeparator && firstDayOfWeekendIndex === getDay(date)
-            }
-            weekendEnd={
-              showWeekendSeparator && lastDayOfWeekendIndex === getDay(date)
-            }
-          >
-            <DateComponent
-              date={date}
-              modifiers={dateModifiers}
-              aria-label={formatDateFull(date)}
-              onClick={onDateClick}
-              onDateKeyDown={onDateKeyDown}
-              preventKeyboardFocus={preventKeyboardFocus}
-              isKeyboardFocusable={isKeyboardFocusable}
-              isFocused={isSameDay(date, focusedDate)}
-              isSelected={isSameDay(date, selectedDate)}
-              isBlocked={
-                minDate && maxDate
-                  ? !isWithinRange(date, minDate, maxDate)
-                  : false
+        {this.props.dates.map(date => {
+          const isBlocked =
+            minDate && maxDate ? !isWithinRange(date, minDate, maxDate) : false;
```

Comment:
Can't this be `minDate && maxDate && !isWithinRange(date, minDate, maxDate)` which avoids the ternary?

Reply:
Yes - I just lifted and shifted how it had been declared by the previous writer when it was declared inline in the prop - can update if preferred?

Reply:
Ah, I didn't realise. Let's keep it as is to reduce the number of changes